scripts/check-gofmt.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eufo pipefail
+#
+# HEIF codec.
+# Copyright (c) 2018 struktur AG, Joachim Bauch <bauch@struktur.de>
+#
+# This file is part of libheif.
+#
+# libheif is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# libheif is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with libheif.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+
+ERR=$(find "$DIR"/.. -name *.go -exec go fmt "{}" \;)
+if [[ (-v ABORT_ON_ERR) && (-n "$ERR") ]]; then
```

Comment:
I think this should be `if [ -n "$ERR" ]; then`? You are not setting `ABORT_ON_ERR` anywhere and the script should fail on errors.

Reply:
Right, fixed.